en/embedding.html

```diff
@@ -312,59 +380,125 @@ <h4 id="colbert-ranking">ColBert ranking</h4>
 }
 </pre>
 
-<h2 id="embedding-a-query-text">Embedding a query text</h2>
-<p>Where you would otherwise supply a tensor in a query request,
-you can with an embedder configured instead supply any text enclosed in <code>embed()</code>, e.g:</p>
 
+
+<h2 id="onnx-export">Exporting HF models to ONNX format</h2>
+
+<p>Transformer based models have named inputs and outputs that needs to be compatible
+with the input and output names used by the embedder.</p>
+
+<p>The <a href="https://github.com/vespa-engine/sample-apps/tree/master/simple-semantic-search#model-exporting">simple-semantic-search</a>
+sample app includes two scripts to export models and vocabulary files using the default expected input and output names
+for embedders using ONNX models. The input and output names to use can also be overriden by the various <code>transformer-</code>
+parameters in the <a href="reference/embedding-reference.html">config of the embedder in question</a>.
+
+<h3 id="onnx-debug">Debugging ONNX models</h3>
```

Comment:
overlaps with https://docs.vespa.ai/en/onnx.html#using-vespa-analyze-onnx-model - maybe the export/debug part here (that are great!) should be in the onnx doc instead, linked from here. I notice this PR does not add this, just rearrange, so can harmonize this in a follow-up PR

Reply:
Yes, agree that should be consolidated elsewhere.